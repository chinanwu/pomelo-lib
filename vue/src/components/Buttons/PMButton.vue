<template>
  <PButton c-class="PMButton">
    <template #content>
      <slot name="content"></slot>
    </template>
  </PButton>
</template>

<script>
import PButton from "@/components/Buttons/PButton";

export default {
  name: "PMButton",
  components: {
    PButton
  }
};
</script>

<style lang="less">
.PMButton {
  font-size: 1.2rem;
}
</style>
