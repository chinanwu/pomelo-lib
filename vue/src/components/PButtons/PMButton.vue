<template>
  <PButton class="PMButton">
    <template #content>
      <slot name="content"></slot>
    </template>
  </PButton>
</template>

<script>
import PButton from "@/components/PButtons/PButton";

export default {
  name: "PMButton",
  components: {
    PButton
  }
};
</script>

<style lang="less" scoped>
.PMButton {
  font-size: 1.6rem;
}
</style>
