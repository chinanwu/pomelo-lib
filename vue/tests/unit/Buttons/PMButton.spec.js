import { shallowMount } from "@vue/test-utils";
import PMButton from "@/components/Buttons/PMButton";

describe("PMButton", () => {
  describe("rendering", () => {
    it("renders PMButton", () => {
      const wrapper = shallowMount(PMButton, {
        slots: {
          content: "Test"
        }
      });
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
