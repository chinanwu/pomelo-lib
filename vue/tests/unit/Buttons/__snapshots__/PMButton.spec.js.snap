// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PMButton rendering renders PMButton 1`] = `<p-button-stub />`;
